src/config.ts

```
@@ -8,22 +8,18 @@ const hyperEvmConfig = defineChain({
   name: "HyperEVM",
   nativeCurrency: {
     decimals: 18,
-    name: "HYPE",
+    name: "HyperEVM",
     symbol: "HYPE",
   },
   rpcUrls: {
     default: {
-      http: [
-        process.env.CHAIN_RPC_URL || "https://hyperliquid-testnet.drpc.org",
-      ],
+      http: [process.env.CHAIN_RPC_URL || ""],
     },
   },
   blockExplorers: {
     default: {
       name: "HyperEVM Explorer",
-      url:
```

We should consider using default RPC URLs instead of empty string fallbacks as if env is not set up it will cause failure on runtime without displaying clear error.Let's stick with default values during development